src/CoreBundle/private/modules/MTHeader/MTHeader.class.php

```diff
@@ -118,11 +118,7 @@ public function &Execute()
             if (TGlobal::CMSUserDefined()) {
                 $this->data['breadcrumb'] = $this->global->GetURLHistory()->GetBreadcrumb(true);
 
-                $lastHistNode = end($this->data['breadcrumb']);
```

Comment:
the "last hist node" is obsolete anyway?

Reply:
It was only used for the following - now replaced - lines.